volatility3/framework/plugins/linux/sockstat.py

```diff
@@ -0,0 +1,506 @@
+# This file is Copyright 2021 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+import logging
+from typing import Callable, Tuple, List, Dict
+
+from volatility3.framework import renderers, interfaces, exceptions, constants, objects
+from volatility3.framework.configuration import requirements
+from volatility3.framework.interfaces import plugins
+from volatility3.framework.objects import utility
+from volatility3.framework.symbols import linux
+from volatility3.plugins.linux import lsof
+
+
+vollog = logging.getLogger(__name__)
+
+class SockHandlers(interfaces.configuration.VersionableInterface):
+    """Handles several socket families extracting the sockets information."""
+
+    _required_framework_version = (2, 0, 0)
+
+    _version = (1, 0, 0)
+
+    def __init__(self, vmlinux, task):
```

Comment:
Typing information would be handy here, so we know whether `vmlinux` is supposed to be a module or a symbol table, or just a string.